includes/REST_API/Stories_Controller.php

```diff
@@ -178,6 +178,7 @@ public function get_item_schema() {
 
 		$schema['properties']['story_data'] = [
 			'description' => __( 'Story data stored as a JSON object. Stored in post_content_filtered field.', 'web-stories' ),
+			'type'        => 'object',
```

Comment:
If type is added, then properties also be added.

Reply:
Why is this needed?

Reply:
Because it's giving a PHP undefined index warning if there is no type set.

Reply:
I don't see the properties being mandatory for the `object` type (not used everywhere in WP REST API either), only the `type` seems to be required since WP assumes it's set.
Can you point me to the information about `properties` being required and `type` not?